tests/utils.py

```diff
@@ -131,6 +131,14 @@ def get_parameter_status(self, parameter):
         else:
             raise NotImplementedError()
 
+class MockMySQLConnection(object):
```

Comment:
Lets call this MockPyMysqlConnection? As we have named the other one as MockPsycopgConnection.